src/components/colorPicker/ColorPickerDialog.tsx

```diff
@@ -338,9 +338,9 @@ class ColorPickerDialog extends PureComponent<Props, State> {
         centerH
         onDismiss={this.onDismiss}
         containerStyle={styles.dialog}
-        panDirection={PanningProvider.Directions.DOWN}
+        direction={PanningProvider.Directions.DOWN}
         testID={`${testID}.dialog`}
-        supportedOrientations={['portrait', 'landscape', 'landscape-left', 'landscape-right']} // iOS only
+        modalProps={{supportedOrientations: ['portrait', 'landscape', 'landscape-left', 'landscape-right']}} // iOS only
```

Comment:
Can we make this a `const` please?

Reply:
Not really, there's a type issue here since this is not a string array per se

Reply:
I think it's best to add it as a const and cast it:
```
import {ModalProps} from '../../components/modal';
...
const MODAL_PROPS = {supportedOrientations: ['portrait', 'landscape', 'landscape-left', 'landscape-right']} as ModalProps;
```